content/docs/use-cases/gitops-for-kubeflow.md

```diff
@@ -28,36 +28,35 @@ Follow the [argo cd getting started guide](https://github.com/argoproj/argo-cd/b
 * For example, if your local machine is OSX and you want to deploy Argo CD and Kubeflow to the same GKE cluster, you could run:
 
 1. Install Argo CD to the Kubernetes Cluster
-```shell
+```
 ARGO_CD_LATEST=$(curl --silent "https://api.github.com/repos/argoproj/argo-cd/releases/latest" | grep '"tag_name"' | sed -E 's/.*"([^"]+)".*/\1/')
 kubectl create namespace argocd
 kubectl apply -n argocd -f https://raw.githubusercontent.com/argoproj/argo-cd/$ARGO_CD_LATEST/manifests/install.yaml
 ```
 
-1. Install Argo CD Cli
-```shell
-brew install argoproj/tap/argocd
-```
-2. For Ubuntu
-```
-wget https://github.com/argoproj/argo-cd/releases/download/v1.2.3/argocd-linux-amd64
-mv argocd-linux-amd64 argocd
-chmod u+x argocd
-export PATH=$PATH:$HOME/path/to/argocd
-```
+2. Install the CLI for Argo CD CLI
+      * On MacOS: 
+        ``` $ brew install argoproj/tap/argocd```
+      * On Ubuntu:
+	```
+           $ wget https://github.com/argoproj/argocd/releases/download/v1.2.3/argocd-linux-amd64
+	   $ mv argocd-linux-amd64 argocd
+	   $ chmod u+x argocd
+	   $ export PATH=$PATH:$HOME/path/to/argocd
+        ```
 
-1. Set extra permissions for Argo CD
+3. Set extra permissions for Argo CD
```

Comment:
The list formatting is still broken here. It's not easy! The three ticks that give the code formatting in the above code block need to be at the same indentation level as the code.

Also, indent the other code blocks in the list items below this line to fix the further list formatting.